support/account_creation_report.sh

```diff
@@ -11,9 +11,7 @@ fi
 
 # Note that although the query below specifies shard_0, the function itself queries and aggregates results across all shards
 if command -v hab > /dev/null 2>&1; then
-  hab pkg exec core/postgresql psql -U hab -q -c "COPY (SELECT * FROM shard_0.account_creation_report('$1')) TO '$output' WITH (FORMAT CSV, HEADER TRUE, FORCE_QUOTE *)" builder_sessionsrv
-
-  if [ $? -eq 0 ]; then
+  if hab pkg exec core/postgresql psql -U hab -q -c "COPY (SELECT * FROM shard_0.account_creation_report('$1')) TO '$output' WITH (FORMAT CSV, HEADER TRUE, FORCE_QUOTE *)" builder_sessionsrv; then
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2181#check-exit-code-directly-with-eg-if-mycmd-not-indirectly-with-

Reply:
Seems fine?  I'm a bash programming novice, and the scripts I write no doubt reflect that, so I'm happy to change these things.  😆